src/components/TodoItem.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import styles from "../styles/TodoItem.module.css";
+import PropTypes from "prop-types";
+
+const TodoItem = ({ itemProp, handleChange, delTodo, setUpdate }) => {
+  const [editing, setEditing] = useState(false);
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const completedStyle = {
+    fontStyle: "italic",
+    color: "#595959",
+    opacity: 0.4,
+    textDecoration: "line-through",
+  };
+
+  let viewMode = {};
+  let editMode = {};
+  if (editing) {
+    viewMode.display = "none";
+  } else {
+    editMode.display = "none";
+  }
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === "Enter") {
+      setEditing(false);
+    }
+  };
+
+  return (
+    <li className={styles.item}>
+      <div className={styles.content} style={viewMode}>
+        <input
+          type="checkbox"
+          checked={itemProp.completed}
+          onChange={() => handleChange(itemProp.id)}
+        />
+        <button onClick={handleEditing}>Edit</button>
+        <button onClick={() => delTodo(itemProp.id)}>Delete</button>
```

Comment:
Adding the "type" attribute in HTML is a great way to ensure proper validation and optimal performance of web pages. It is a small but significant step toward creating well-structured and accessible web content😃